net/faq/_index.md

```diff
@@ -5,10 +5,518 @@ type: docs
+
+```csharp
+// Create a new PDF document
+using (Document pdfDoc = new Document("input.pdf"))
+{
+    // Create a TextFragmentAbsorber with a regex pattern
+    TextFragmentAbsorber absorber = new TextFragmentAbsorber(@"\b\w+\b", new TextSearchOptions(true));
+
+    // Process the PDF document
+    absorber.Visit(pdfDoc);
+
+    // Loop through matched fragments
+    foreach (TextFragment fragment in asorber.TextFragments)
+    {
+        // Example: Change text color of matched fragments
+        fragment.TextState.ForegroundColor = Color.Red;
+        
+        // Example: Modify the matched text if needed
+        fragment.Text = "[REDACTED]";  // Replace with custom text
+    }
+
+    // Save the PDF document
+    pdfDoc.Save("output.pdf");
+}
+```
+
+The key points are:
+- Pass a regular expression pattern to the `TextFragmentAbsorber` constructor.
+- The pattern `@"\b\w+\b"` will match whole words.
+- Call the `Visit()` method to process the PDF document.
+- Access the extracted `TextFragments` objects from the `TextFragments` property.
+        </p>
+    </div>
+
+    <div itemscope itemprop="mainEntity" itemtype="https://schema.org/Question" class="faq-question">
+        <h3 itemprop="name" class="faq-q">
+How to make a valid PDF/A document unless the missing font or its substitution is provided?
+        </h3>
+    </div>
+    <div itemscope itemprop="acceptedAnswer" itemtype="https://schema.org/Answer">
+        <p itemprop="text" class="faq-a">
+To create a valid PDF/A document in Aspose.PDF for .NET, you need to ensure that all required fonts are embedded or substituted. Here's an example:
+
+- Use `Convert` to ensure the document meets PDF/A standards.
+
+```csharp
+// Load the document
+using (Document pdfDoc = new Document("input.pdf"))
+{
+    PdfFormatConversionOptions options = new PdfFormatConversionOptions("conversion-log.xml", PdfFormat.PDF_A_1B, ConvertErrorAction.Delete);
+
+    // Substitute all missing fonts with the default one
+    options.FontEmbeddingOptions.UseDefaultSubstitution = true;
+
+    // Convert to PDF/A
+    pdfDoc.Convert(options);
+
+    // Save as PDF/A
+    pdfDoc.Save("output-pdfa.pdf");
+}
+```
+        </p>
+    </div>
+
+    <div itemscope itemprop="mainEntity" itemtype="https://schema.org/Question" class="faq-question">
+        <h3 itemprop="name" class="faq-q">
+I see errors in PDF/A conversion log. Does it mean that the document wasn't converted successfully?
+        </h3>
+    </div>
+    <div itemscope itemprop="acceptedAnswer" itemtype="https://schema.org/Answer">
+        <p itemprop="text" class="faq-a">
+No, Aspose.PDF logs all problems it encountered, including the ones that were automatically fixed. If all entries in the log are marked as Convertable="True", it means that all problems were fixed, and the document was successfully converted. Only the entries with Convertable=False indicate the conversion failure.
```

Comment:
If all entries in the log are marked as Convertable="True, " all problems were fixed, and the document was successfully converted. Only the entries with Convertable=False indicate the conversion failure.